docs/team/kangzongxian.md

```diff
@@ -5,36 +5,55 @@ title: Kang Zong Xian's Project Portfolio Page
 
 ### Project: GuestList
 
-GuestList is a one stop access to get all the necessary guest data.
+GuestBook is a **desktop app for managing guests in a hotel,
+optimized for use via a Command Line Interface** (CLI)
+while still having the benefits of a Graphical User Interface (GUI).
+If you are a hotel manager who can type fast, GuestBook can get your
+guest management tasks done faster than traditional GUI apps.
 The user interacts with it using a CLI, and it has a GUI created with JavaFX.
 It is written in Java, and has about 10 kLoC.
 This project is based off AddressBook - Level 3.
 
 Given below are my contributions to the project.
 
-* **New Feature**:
-    * to be added soon
+* **New Feature**: Add Room Field
+
+* **New Feature**: Make Room Field unique to each Guest
+  * Ensures that there are no guests staying in the same room
+  * This feature greatly aids in the precision of GuestBook, as unrelated guests are unlikely to 
+  stay in the same room. This also prevents the user from accidentally entering a room that is occupied.
+  
 
 * **Code contributed**: [[RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2022-09-16&tabOpen=true&tabType=authorship&zFR=false&tabAuthor=kangzongxian&tabRepo=AY2223S1-CS2103T-W16-1%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)]
 
-* **Project management**:
-    * to be added soon
 
 * **Enhancements to existing features**:
-    * to be added soon
+    * Removed Address Field and Tag Field to fit the GuestBook specificity.
+      * The Address Field and Tag Field are not necessary in GuestBook, 
+      and removing them significantly improves the clarity of our codebase.
 
 * **Documentation**:
     * User Guide:
-        * to be added soon
+        * Did final checks to ensure consistency of UG (pull request #140).
+        * Added How to use the User Guide in user guide (pull request #132).
 
     * Developer Guide:
-        * to be added soon
+        * Did final checks to ensure consistency of DG (pull request #140).
+        * Added implementation of find command in DG (pull request #114).
+        * Add UML Diagrams (pull request #114)
 
 * **Contributions to team-based tasks**:
-    * to be added soon
+    * Facilitated and tidied up contents of postmortem v1.2
+
+**Project management**:
+* Used Github issue tracker.
+* Followed the branching workflow.
+* Contributed to meetings by taking down minutes and facilitating discussions.
 
 * **Review/mentoring contributions**:
-    * to be added soon
+  * Team members' PR reviews (with non-trivial review comments): pull request #191, #142, #138, #131, #119, #111
+    , #109, #73, #68, #65, #58
```

Comment:
```suggestion
  * Team members' PR reviews (with non-trivial review comments): pull request #194, #193, #152, #147, #146, #145, #140, #137, #132, #130, #124, #114, #102, #79, #74, #70, #55, #54
```

Reply:
I think the PR's you mention are the ones I created and not the ones I reviewed, which I think is what the statement wants!